0004-ledger-plugin-interface/0004-ledger-plugin-interface.md

```diff
@@ -199,82 +204,83 @@ Submit a fulfillment to a ledger. Plugin must be connected, otherwise the promis
 
 **TODO**: Define what the message format is.  Plugin must be connected, otherwise the promise should reject.
 
-#### Event: `receive`
-<code>ledgerPlugin.on('receive', ( **transfer**:[IncomingTransfer](#incomingtransfer) ) ⇒ )</code>
+### Event: `*_transfer`
+<code style="">ledgerPlugin.on('incoming_transfer',
+  (
+    **transfer**:[Transfer](#class-transfer),
+  ) ⇒
+)</code>
 
-Emitted when a transfer is received.
+Emitted on an outgoing/incoming transfer which does not have a condition. 
```

Comment:
"on" is a bit vague. Why not "Emitted after an outgoing/incoming transfer with no condition has been executed on the ledger."